app/cli/internal/action/attestation_add.go

```diff
@@ -81,7 +81,21 @@ func (action *AttestationAdd) Run(k, v string) error {
 
 	cc := casclient.New(artifactCASConn, casclient.WithLogger(action.Logger))
 
-	if err := action.c.AddMaterial(k, v, cc); err != nil {
+	// // TODO: retrieve this information from backend
+	backend := &casclient.CASBackend{
+		Uploader: cc,
+		Name:     "OCI",
+		// 100MB max size
+		MaxSize: 100 * 1024 * 1024,
+	}
+
```

Comment:
Do we need this comment here?

Reply:
What comment? the 100MB? or the commented out inline backend?

Reply:
the commented one about 500KB

Reply:
It will be gone soon.